wurst/dummy/InstantDummyCaster.wurst

```diff
@@ -16,51 +16,55 @@ public class InstantDummyCaster
 	
```

Comment:
Belongs into it's own PR?

Reply:
It should be, probably... Though this PR is almost meaningless without it.